Ginger/Ginger/ALM/Repository/OctaneRepository.cs

```diff
@@ -75,27 +86,117 @@ public override List<string> GetTestPlanExplorer(string path)
         {
-            throw new NotImplementedException();
+            if (selectedTestSets != null && selectedTestSets.Count() > 0)
+            {
+                ObservableList<QCTestSetTreeItem> testSetsItemsToImport = new ObservableList<QCTestSetTreeItem>();
+                foreach (QCTestSetTreeItem testSetItem in selectedTestSets)
+                {
+                    //check if some of the Test Set was already imported                
+                    if (testSetItem.AlreadyImported == true)
+                    {
+                        Amdocs.Ginger.Common.eUserMsgSelection userSelection = Reporter.ToUser(eUserMsgKey.TestSetExists, testSetItem.TestSetName);
+                        if (userSelection == Amdocs.Ginger.Common.eUserMsgSelection.Yes)
+                        {
+                            //Delete the mapped BF
+                            File.Delete(testSetItem.MappedBusinessFlow.FileName);
+                            testSetsItemsToImport.Add(testSetItem);
+                        }
+                    }
+                    else
+                    {
+                        testSetsItemsToImport.Add(testSetItem);
+                    }
+                }
+
+                if (testSetsItemsToImport.Count == 0) return false; //noting to import
+
+                //Refresh Ginger repository and allow GingerQC to use it                
+                ALMIntegration.Instance.AlmCore.GingerActivitiesGroupsRepo = WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<ActivitiesGroup>();
+                ALMIntegration.Instance.AlmCore.GingerActivitiesRepo = WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<Activity>();
+
+                foreach (QCTestSetTreeItem testSetItemtoImport in testSetsItemsToImport)
+                {
+                    try
+                    {
+                        //import test set data
+                        Reporter.ToStatus(eStatusMsgKey.ALMTestSetImport, null, testSetItemtoImport.TestSetName);
+                        GingerCore.ALM.QC.QCTestSet TS = new GingerCore.ALM.QC.QCTestSet();
+                        TS.TestSetID = testSetItemtoImport.TestSetID;
+                        TS.TestSetName = testSetItemtoImport.TestSetName;
+                        TS.TestSetPath = testSetItemtoImport.Path;
+                        TS = ((OctaneCore)ALMIntegration.Instance.AlmCore).ImportTestSetData(TS);
+
+                        //convert test set into BF
+                        BusinessFlow tsBusFlow = ((OctaneCore)ALMIntegration.Instance.AlmCore).ConvertQCTestSetToBF(TS);
+
+                        if (WorkSpace.Instance.Solution.MainApplication != null)
+                        {
+                            //add the applications mapped to the Activities
+                            foreach (Activity activ in tsBusFlow.Activities)
+                                if (string.IsNullOrEmpty(activ.TargetApplication) == false)
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6317725)